runtime/doc/vim_diff.txt

```diff
@@ -213,24 +213,28 @@ Additional differences:
 
 - |shada-c| has no effect.
 - |shada-s| now limits size of every item and not just registers.
-- When reading ShaDa files items are merged according to the timestamp.
-  |shada-merging|
 - 'viminfo' option got renamed to 'shada'. Old option is kept as an alias for
   compatibility reasons.
 - |:wviminfo| was renamed to |:wshada|, |:rviminfo| to |:rshada|.  Old
   commands are still kept.
-- When writing (|:wshada| without bang or at exit) it merges much more data,
-  and does this according to the timestamp.  Vim merges only marks.
-  |shada-merging|
 - ShaDa file format was designed with forward and backward compatibility in
   mind. |shada-compatibility|
 - Some errors make ShaDa code keep temporary file in-place for user to decide
   what to do with it.  Vim deletes temporary file in these cases.
   |shada-error-handling|
-- Vim keeps no timestamps at all, neither in viminfo file nor in the instance
-  itself.
 - ShaDa file keeps search direction (|v:searchforward|), viminfo does not.
 
+Lua interface (|if_lua.txt|):
```

Comment:
With the current code I can resurrect `vim.buffer`, `vim.window`, `vim.command`, `vim.eval`, `vim.line`, `vim.open`: purely in lua. And create something which will look like what `vim.funcref` should look like (though I would rather implement this on top of references). But `vim.dict`, `vim.list` and `vim.type` are out or need to be implemented in C (basically copy implementation from Vim).

Not sure about `vim.beep()`: there are many indirect/hacky ways to get a beep, but I do not know VimL or API function to get a beep directly. Though with my setup it may just as well be `function() end`, so I never bothered with learning something as useless as trying to emit a beep.